omegaconf/basecontainer.py

```diff
@@ -222,9 +220,7 @@ def convert(val: Node) -> Any:
                 node = conf._get_node(key)
                 assert isinstance(node, Node)
                 if resolve:
-                    node = node._dereference_node(
-                        throw_on_missing=False, throw_on_resolution_failure=True
-                    )
+                    node = node._dereference_node(throw_on_resolution_failure=True)
```

Comment:
hmm, I wonder if `__str__` should never throw.

Reply:
This is in `_to_content()`, what's the link with `__str__`?

Reply:
> The link is that GitHub diff view made it seem it like was `__str__()`.
Oh ok, resolving then :)